MANIFEST.in

```diff
@@ -0,0 +1,37 @@
+recursive-include unionml/templates *
+
+include README.md
+include src/pythae/py.typed  # marker file for PEP 561
+
+include CHANGELOG.md
+include changelog.md
```

Comment:
No such file in repo

Reply:
I did not see any changelog. So want to ask you if would like to maintain a changelog file, where changes before a release are accumulated and finally made part of the release notes. 

If not, I will drop these lines.

Reply:
For now I do not intend to have a changelog file so I think we can drop those lines

Reply:
Done